examples/pytorch/image-classification/README.md

```diff
@@ -197,7 +198,7 @@ accelerate test
 that will check everything is ready for training. Finally, you can launch training with
 
 ```bash
-accelerate launch run_image_classification_trainer.py
+accelerate launch run_image_classification_no_trainer.py --image_column_name img
```

Comment:
the default should not need this no?

Reply:
I set the default to `image` because that's the name of the image column for most image datasets. But Cifar10, which is used in this example, has it named `img` for some reason.

Reply:
got it thanks